homeassistant/components/starline/account.py

```diff
@@ -141,7 +142,9 @@ def device_info(device: StarlineDevice) -> DeviceInfo:
     def gps_attrs(device: StarlineDevice) -> dict[str, Any]:
         """Attributes for device tracker."""
         return {
-            "updated": datetime.utcfromtimestamp(device.position["ts"]).isoformat(),
+            "updated": dt_util.utc_from_timestamp(device.position["ts"])
+            .replace(tzinfo=None)
+            .isoformat(),
```

Comment:
I guess we could do here:
```suggestion
            `.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```
Would need to check if there are considerations regarding the fractional part.

Reply:
For the UI we need the output of `isoformat()`.